sdk/eventhub/azure-eventhub/azure/eventhub/_client_base.py

```diff
@@ -124,11 +130,22 @@ def _parse_conn_str(
     # check that endpoint is valid
     if not endpoint:
         raise ValueError("Connection string is either blank or malformed.")
+
     parsed = urlparse(endpoint)
     if not parsed.netloc:
         raise ValueError("Invalid Endpoint on the Connection String.")
     host = cast(str, parsed.netloc.strip())
 
+    emulator = bool(use_emulator=="true")
```

Comment:
do we need to lower on this or will it always be "true"? i.e.
```suggestion
    emulator = use_emulator.lower()=="true"
```